react/features/stream-effects/blur/index.js

```diff
@@ -23,14 +27,12 @@ export function createBlurEffect() {
         return Promise.reject(new Error('JitsiStreamBlurEffect not supported!'));
     }
 
-    // change the TF backend to use WebGL
-    TF.setBackend('webgl').then(() => {
-        logger.info('TensorFlow backend changed to use the WebGL');
-    })
-    .catch(error => {
-        logger.info('TensorFlow backend could not be changed to use WebGL, performance may suffer: ',
-            error);
-    });
+    tfc.setBackend(webGlBackend, true);
```

Comment:
Where can I read about the second argument? Before you were using a promise version but now you're using a sync version?

Reply:
https://js.tensorflow.org/api/0.6.1/#setBackend. The second argument starts TF in safe mode which forces it to clean the intermediate tensors automatically.